quasar_entities/src/ledger.rs

```diff
@@ -1,11 +1,13 @@
-//! `SeaORM` Entity. Generated by sea-orm-codegen 0.12.2
-
+use async_graphql::ComplexObject;
 use sea_orm::{entity::prelude::*, Set};
+
 use stellar_node_entities::ledgerheaders;
 use stellar_xdr::{Error, LedgerHeader, ReadXdr};
 
-#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq, async_graphql::SimpleObject)]
 #[sea_orm(table_name = "ledgers")]
+#[graphql(complex)]
+#[graphql(name = "Ledgers")]
 pub struct Model {
```

Comment:
Could we rename entities from `Model` to a representative name eg Ledger?

Reply:
This seems to be the naming convention in SeaORM. The original of this file was actually generated by SeaORM CLI tooling. There's also `ActiveModel` and `Entity` that are generated by the macros. For now I'm not familiar enough with SeaORM to change the default naming conventions.